packages/reaction-router/common/init.js

```diff
@@ -88,7 +88,7 @@ const getRegistryRouteName = (packageName, registryItem) => {
  */
 ReactionRouter.initPackageRoutes = () => {
   const pkgs = ReactionCore.Collections.Packages.find().fetch();
-  const prefix = ReactionCore.getShopName().toLowerCase(); // todo add shopId
+  const prefix = getSlug(ReactionCore.getShopName().toLowerCase()); // todo add shopId
```

Comment:
Except you know, if somebody had to create a `return_url` for something. Just speculating.